styles.css

```diff
@@ -0,0 +1,470 @@
+  align-items: center;
+  box-shadow: 0.2rem 0.2rem 0 rgba(0, 0, 0, 0.34);
+  margin: 0 0.25rem;
+  display: flex;
+  overflow: hidden;
+}
+
+.toolBar-button:hover {
+  background-color: #454545;
+}
+.toolBar-button .shading-toggle-buttons {
+  margin: 0;
+}
+div.toolBar-button.shading-toggle-buttons:hover {
+  background-color: #282828;
+}
+
+.toolBar-shading-toggle,
+.toolBar-gizmo-toggle {
+  display: none;
+}
+.toolBar-shading-option {
+  border-right: 1px solid #181818;
+  background-color: #545454;
+}
+
+.toolBar-shading-toggle
+  input[type="radio"]:nth-child(1):checked
+  ~ .shading-toggle-buttons
+  label:nth-child(1),
+.toolBar-shading-toggle
+  input[type="radio"]:nth-child(2):checked
+  ~ .shading-toggle-buttons
+  label:nth-child(2),
+.toolBar-shading-toggle
+  input[type="radio"]:nth-child(3):checked
+  ~ .shading-toggle-buttons
+  label:nth-child(3),
+.toolBar-shading-toggle
+  input[type="radio"]:nth-child(4):checked
+  ~ .shading-toggle-buttons
+  label:nth-child(4) {
+  background-color: #4772b3;
+}
+
+.gizmo-toggle input[type="checkbox"]:nth-child(1):checked ~ .gizmo-icon {
+  background-color: #4772b3;
+}
+.gizmo-reveal-image {
+  display: none;
+}
+
+.gizmo-toggle
+  input[type="checkbox"]:nth-child(1):checked
+  ~ .gizmo-reveal-image {
+  display: block;
+}
+
+.toolBar-shading-toggle
+  input[type="radio"]:nth-child(1):checked
+  ~ #rendered-goat-container
+  div:nth-child(1),
+.toolBar-shading-toggle
+  input[type="radio"]:nth-child(2):checked
+  ~ #rendered-goat-container
+  div:nth-child(2),
+.toolBar-shading-toggle
+  input[type="radio"]:nth-child(3):checked
+  ~ #rendered-goat-container
+  div:nth-child(3),
+.toolBar-shading-toggle
+  input[type="radio"]:nth-child(4):checked
+  ~ #rendered-goat-container
+  div:nth-child(4) {
+  display: block;
+}
+
+.icon-link {
+  text-decoration: none;
+}
+
+.icon-text {
+  font-size: 1.25rem;
+  background-color: #1f1f1f;
+  padding: 0.2rem 1.25rem;
+  height: 2rem;
+}
+ul.navBarList {
+  display: block;
+  margin: 0;
+  text-align: center;
+  list-style-type: none;
+  padding: 0;
+}
+li.navBarTab {
+  display: inline-block;
+}
+
+.navBarList a:hover {
+  background-color: #777;
+  color: #222;
+}
+.navBarTab a {
+  display: block;
+  color: #969696;
+  font-size: 1.2rem;
+  line-height: 1.6em;
+  background-color: #232323;
+  padding: 0 0.5em;
+  margin: 0.25em 0.2em 0;
+  border-radius: 0.75em 0.75em 0 0;
+}
+li.navBarTab.navBarCurrentPage a {
+  background-color: #303030;
+}
+.navBarTab.navBarCurrentPage a {
+  color: #f2f2f2;
+}
+@media (min-width: 768px) {
+  .navBar {
+    grid-template-columns: 1fr 400px 1fr;
+  }
+  .navDecoration {
+    display: flex;
+    align-items: center;
+    padding: 0.25rem;
+  }
+  .navBarTab a {
+    font-size: 1.5rem;
+  }
+}
+
+@media (min-width: 900px) {
+  .toolBar-button.extraDecoration {
+    display: flex;
+  }
+}
+@media (min-width: 1300px) {
+  .toolBar-shading-toggle,
+  .toolBar-gizmo-toggle {
+    display: flex;
+  }
+}
```

Comment:
Same comment as in the HTML, don't do both `camelCase` and `kabob-case`, pick one.